test/integration-test.py

```diff
@@ -315,15 +315,30 @@ def check_slow_queries():
     """Checks that we haven't run any slow queries during the integration test.
 
     This depends on flags set on mysqld in docker-compose.yml.
+
+    We skip the boulder_sa_test database because we manually run a bunch of
+    non-indexed queries in unittests. We skip actions by the setup and root
+    users because they're known to be non-indexed. Similarly we skip the
+    cert_checker, mailer, and janitor's work because they are known to be
+    slow (though we should eventually improve these).
+    The SELECT ... IN () on the authz2 table shows up in the slow query log
+    a lot. Presumably when there are a lot of entries in the IN() argument
+    and the table is small, it's not efficient to use the index. But we
+    should dig into this more.
```

Comment:
Can you open Boulder issues for these outstanding optimization problems? I think they're likely to be forgotten otherwise.

Reply:
I opened one issue for all of them instead: https://github.com/letsencrypt/boulder/issues/4583 We can carve off smaller issues on a per-sprint basis.